x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,94 @@ export const ModelDeploymentInProgress = ({
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="checkInCircleFilled" />
+            </EuiFlexItem>
+            <EuiFlexItem grow>
+              <EuiTitle size="xs">
+                <h4>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedTitle"
+                    defaultMessage="Your ELSER model has deployed but not started."
+                  />
+                </h4>
+              </EuiTitle>
+            </EuiFlexItem>
+            {isDismissable && (
+              <EuiFlexItem grow={false}>
+                <TextExpansionDismissButton dismiss={dismiss} />
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiText>
+            <FormattedMessage
+              id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.deployedBody"
+              defaultMessage="You may start the model in a single-threaded configuration for testing, or tune the performance for a production environment."
+            />
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem>
+              <EuiButton
+                aria-label={i18n.translate(
+                  'xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.startModelButton',
+                  { defaultMessage: 'Start ELSER model' }
+                )}
+                color="success"
+                disabled={isStartButtonDisabled}
+                iconType="playFilled"
+                onClick={() => startTextExpansionModel(undefined)}
+              >
+                {i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.textExpansionCallOut.startModelButton.label',
+                  {
+                    defaultMessage: 'Start single-threaded',
+                  }
+                )}
+              </EuiButton>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiButtonEmpty
+                aria-label={i18n.translate(
+                  'xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.fineTuneModelButton',
+                  { defaultMessage: 'Fine-tune ELSER model' }
+                )}
+                iconSide="left"
+                iconType="wrench"
+                onClick={() =>
+                  KibanaLogic.values.navigateToUrl('/app/ml/trained_models', {
+                    shouldNotCreateHref: true,
+                  })
+                }
+              >
+                {i18n.translate('xpack.enterpriseSearch.content.engine.api.step1.viewKeysButton', {
+                  defaultMessage: 'Fine-tune performance',
+                })}
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+            <EuiFlexItem grow />
```

Comment:
question: can this empty flexitem be deleted?

Reply:
@sphilipse I added it so that the two buttons are left aligned, like in the design:
![Screenshot 2023-04-28 at 10 19 10 AM](https://user-images.githubusercontent.com/14224983/235173086-42bb24ce-2810-4958-8df9-80b81859f6f4.png)

If I remove it, the buttons will fill the horizontal space:
![Screenshot 2023-04-28 at 10 18 25 AM](https://user-images.githubusercontent.com/14224983/235173156-5b62083d-e97e-40a5-82d5-3e73aeadcaf9.png)

Let me know the best practice - is it preferable to fill the space, or if not, is there another way to align the buttons to the left and leave some padding on the right?

Reply:
You don't need to add an empty div for that. You can set `grow={false}` on both buttons, and then add `justifyContent="flexStart"` to the wrapping `EuiFlexGroup`.

Reply:
👍  That works - I made the changes and removed the empty `EuiFlexItem`.